data/35e/wizards_of_the_coast/supplement/dragon_magic/dm_abilities.lst

```diff
@@ -130,11 +130,15 @@ Devour Spell	KEY:Spelleater ~ Devour Spell			CATEGORY:Special Ability	TYPE:Spell
+#Scales AC bonus does not follow a mathematical progression.  - applying with PREVAR instead
+Scales				KEY:Dragonfire Adept ~ Scales					CATEGORY:Special Ability	TYPE:Extraordinary.Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 2nd level, your skin becomes thick and scaly, granting you a +2 bonus to your natural armor. The scales can be of any color or metallic hue; they are often (but not always) of a draconic hue that matches your outlook and alignment. This bonus improves to +3 at 8th level, to +4 at 13th level, and to +5 at 18th level. If you already have a natural armor bonus, use the higher of the two values.		DEFINE:ScalesArmorBonus|0	BONUS:COMBAT|AC|ScalesArmorBonus|TYPE=NaturalArmor		BONUS:VAR|ScalesArmorBonus|2		BONUS:VAR|ScalesArmorBonus|1|PREVARGTEQ:DragonfireAdeptLVL,8	BONUS:VAR|ScalesArmorBonus|1|PREVARGTEQ:DragonfireAdeptLVL,13	BONUS:VAR|ScalesArmorBonus|1|PREVARGTEQ:DragonfireAdeptLVL,18	
 Dragonkin				KEY:Dragonfire Adept ~ Dragonkin				CATEGORY:Special Ability	TYPE:Extraordinary.Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 4th level, you gain a +4 competence bonus on Diplomacy checks made to influence the attitude of dragons or creatures of the dragonblood subtype (see page 4). You are treated as a dragon for the purpose of determining whether frightful presence can affect you.
-Damage Reduction			KEY:Dragonfire Adept ~ Damage Reduction			CATEGORY:Special Ability	TYPE:Extraordinary.Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 6th level, you gain damage reduction 2/magic. At 16th level, this improves to damage reduction 5/magic.
+#Fixed DR 
+Damage Reduction			KEY:Dragonfire Adept ~ Damage Reduction			CATEGORY:Special Ability	TYPE:Extraordinary.Dragonfire Adept Class Feature.SpecialQuality	SOURCEPAGE:p.26	DESC:At 6th level, you gain damage reduction 2/magic. At 16th level, this improves to damage reduction 5/magic.		DR:2/magic		ABILITY:Special Ability|AUTOMATIC|Dragonfire Adept ~ Beter Damage Reduction|PREVARGTEQ:PREVARGTEQ:DragonfireAdeptLVL,16
```

Comment:
Dragonfire Adept ~ Beter Damage Reduction should be
Dragonfire Adept ~ Better Damage Reduction
|PREVARGTEQ:PREVARGTEQ:DragonfireAdeptLVL,16 should be
|PREVARGTEQ:DragonfireAdeptLVL,16

BONUS:DR|Magic|3|PREVARGTEQ:DragonfireAdeptLVL,16 would be less code and net the correct result